ubuntudesign/vanilla-framework
# Added small screen vertical divider

## Done
- Added small screen styles of vertical divider
- Removed the unused `_verticial-divider` component as it is now a modifier of equal-heights
- Updated the comment examples to use the recommended classes

## QA
- Run `gulp build`
- Check the vertical-divider's in the demo looks the same on large screens
- On small screens they should stack with dividers between them

## Details
Fixes https://github.com/ubuntudesign/www.ubuntu.com/issues/187
